### PR TITLE
docs: add documentation about scaling

### PR DESCRIPTION
**Pull request by fd0r**

### CODE REVIEW

File: docs/advanced_examples/utils/scaling_comparison_utils.py
Comment (RomanBredehoft) [inline docs/advanced_examples/utils/scaling_comparison_utils.py:1]: not sure what this file is about, it does not seem to be used anywhere !

File: docs/advanced_examples/LogisticRegression.ipynb
Comment (RomanBredehoft) [inline docs/advanced_examples/LogisticRegression.ipynb:460]: this section is awesome ! I would just add a title to it so that we quickly get what the following is about

**Review by RomanBredehoft — CHANGES_REQUESTED** [review]
awesome thanks ! I would just add a title in the notebook for clarity

File: docs/built-in-models/linear.md
Comment (bcm-at-zama) [inline docs/built-in-models/linear.md:31]: - do we mention using **Scaler directly in the doc? I think it would be interesting to have code block directly with one Scaler example
- I think it would be interesting that also, we have the word 'normalization', such that it's searchable in the doc

Comment (fd0r) [reply docs/built-in-models/linear.md:31]: The goal here is really to make the user take a look at the notebook such that it's clear what the impact of it can be and  they can directly copy-paste the snippet from there.

Could make sense to also have a snippet here but as you want

Comment (bcm-at-zama) [reply docs/built-in-models/linear.md:31]: I think we need to convince them there is a big problem here, else they'll never go to the nb

**Review by RomanBredehoft — APPROVED** [review]
thanks !

File: docs/built-in-models/linear.md
Comment (fd0r) [reply docs/built-in-models/linear.md:31]: I don't think a code snippet will do that tbh

It might not be an issue for datasets where the features have more or less the same range.

Comment (bcm-at-zama) [reply docs/built-in-models/linear.md:31]: it's not to convince them there is a problem with the code snippet: it's to tell them how to not have the problem with code snippet, and to have Scaler and normalization somewhere in the doc

**Comment by fd0r** [comment]
Waiting for @andrei-stoian-zama 's approval to merge.

File: docs/built-in-models/linear.md
Comment (jfrery) [inline docs/built-in-models/linear.md:31]: Not sure about the 
```
The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to uniformize the distribution of the different features.
```

Maybe something like:
```suggestion
The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to make all feature distribution similar by using standard or min-max normalization.
```

**Review by jfrery — CHANGES_REQUESTED** [review]
Small comment + I had not seen the end of the notebook. I find it very dense Maybe we put the code in a file and have the figure somewhere in the doc instead?

**Comment by fd0r** [comment]
@jfrery so remove from the notebook and just include the figure with some comments in the markdown file?

**Review by andrei-stoian-zama — CHANGES_REQUESTED** [review]
I think it needs a bit more work:

- Introduce the last section
- Remove the utility code or make it smaller
- The graphs need bigger text 
- it's not clear why there is variation in accuracy between no-transform and normalized

File: docs/built-in-models/linear.md
Comment (RomanBredehoft) [inline docs/built-in-models/linear.md:32]: could be worth to include the graphs from the notebook here for better understanding !

Comment (fd0r) [reply docs/built-in-models/linear.md:32]: They might take too much place in the readme, what do you think about it @andrei-stoian-zama ?

Comment (RomanBredehoft) [reply docs/built-in-models/linear.md:32]: yeah indeed, well as you guys wish

**Review by RomanBredehoft — APPROVED** [review]
thanks !

**Review by andrei-stoian-zama — APPROVED** [review]
thanks!

**Comment by fd0r** [comment]
> I still find the end of the notebook a bit intense. Could we just keep the graph + explanation and put the pre processing code outside of the notebook? or add some comments?

Do you mean an explanation per processing done on the features?

**Review by jfrery — APPROVED** [review]

**Pull request merged**
